src/config/mod.rs

```diff
@@ -1800,7 +1800,13 @@ impl Default for RaftEngineConfig {
     fn default() -> Self {
         Self {
             enable: true,
-            config: RawRaftEngineConfig::default(),
+            config: RawRaftEngineConfig {
```

Comment:
Could you move this to validate instead?

Reply:
If we do this, users cannot manually close `enable-log-recycle` in the configuration file.
IMO, in current stage(not release this modification in the `raft-engine` lib), it's better to do this modification at the initialization of `RaftEngineConfig`.